stingray/lightcurve.py

```diff
@@ -221,6 +221,8 @@ def join(self, other):
         objects. The individual count per bin in the resulting object will
         be the sum total of the count per bin in both lightcurve objects.
```

Comment:
This sentence  is misleading. The result count per bin will be the individual count per bin, or the average in case of overlapping.